### ISSUE TITLE
App completely non-functional on Pixel 9 Pro Fold + Android 15

### ISSUE THREAD

**Issue by notmydevice** [babybuddy/babybuddy-for-android#90]
The Play Store build, when installed on Pixel 9 Pro Fold running Android 15, does not function at all.

This is the case both in standard handset form factor, as well as when the device is unfolded.

Upon launching the app, the login page is displayed. But the green "to get started" hint seems to indicate that one or more UI elements are not positioned as expected, perhaps off-screen. Also, the app title bar physically collides with the notification area:

![image](https://github.com/user-attachments/assets/89010be5-7ef0-43bd-a648-26da2594c051)

It is then possible to login. However, that's the last time the user is able to interact with the app. Following login, the following screen is displayed, which does not react to any type of user input:

![image](https://github.com/user-attachments/assets/d6778eb0-52ee-4bf7-bcb1-98c24ac3d56b)

I have only tested on the aforementioned configuration, it's possible this behavior extends to other devices and/or Android versions.

**Comment by MrApplejuice** [comment]
Hey @notmydevice! 

Thanks for the report and sorry for the inconvenience. This is [the result from me updating the SDK compatibility level](https://github.com/babybuddy/babybuddy-for-android/pull/87), a side effect that I did not notice at first over but discovered myself 2 days ago over here https://github.com/babybuddy/babybuddy-for-android/issues/89. The [temporary fix has already been merged into `master`](https://github.com/babybuddy/babybuddy-for-android/pull/88), but still needs to be published. I can push the update tomorrow, it should then be available in the next day or two.

In the mean time you can install the previous version from GitHub as a workaround. Or use it on a android device with a version < 15 if you have access to such a device.

**Comment by pm64** [comment]
Thank you @MrApplejuice for the explanation. Do you have an estimated timeframe for the fix appearing in the latest GitHub release? No worries if not.

**Comment by MrApplejuice** [comment]
All fixes are published now. Is it actually fixed now? I do not have a physical Android 15 device, so I cannot verify.

**Comment by notmydevice** [comment]
Yes @MrApplejuice, everything appears to work as expected now, thank you!

Very sorry to ask you in this thread, I know it's a huge no-no, but there's one other concern that's preventing us from getting up and running. Wondering if you'd mind taking a peek at https://github.com/orgs/babybuddy/discussions/935?
